services/static-webserver/client/source/class/osparc/auth/core/Utils.js

```diff
@@ -20,9 +20,9 @@ qx.Class.define("osparc.auth.core.Utils", {
 
   statics: {
     passwordLengthValidator: function(value, item) {
-      const valid = value != null && value.length > 3;
+      const valid = value != null && value.length > 11;
```

Comment:
beautiful!

Reply:
💅

Reply:
Should that not go to the backend?

Reply:
I was about to edit my comment, I guess that the backend also makes the same check...

Reply:
I will double check that and adapt as well .